src/follow-me/edit.js

```diff
@@ -2,73 +2,107 @@ import { InspectorControls, useBlockProps } from '@wordpress/block-editor';
 import { __ } from '@wordpress/i18n';
 import { useSelect } from '@wordpress/data';
 import { store as coreStore } from '@wordpress/core-data';
-import { SelectControl, PanelBody } from '@wordpress/components';
+import { SelectControl, PanelBody, ToggleControl, TextControl } from '@wordpress/components';
 import { useUserOptions } from '../shared/use-user-options';
 import FollowMe from './follow-me';
 import { useEffect } from '@wordpress/element';
 import { InheritModeBlockFallback } from '../shared/inherit-block-fallback';
 
+/**
+ * Edit component.
+ *
+ * @param {Object} props - Component props.
+ * @param {Object} props.attributes - Block attributes.
+ * @param {Function} props.setAttributes - Set block attributes.
+ * @param {Object} props.context - Block context.
+ * @param {string} props.context.postType - Post type.
+ * @param {number} props.context.postId - Post ID.
+ */
+export default function Edit({
+  attributes,
+  setAttributes,
+  context: {
+	postType,
+	postId,
+  },
```

Comment:
Looks like there's some formatting issues with tabs/spaces and parens spacing in this file

Reply:
Good catch! I've been wrestling with Windsurf to get it to generate with tabs and WP formatting in general

Reply:
ah yes, I gave it a global directive to always use WP coding styles but still have to frequently switch back to PHPStorm and have it prettify the file(s)